tests/src/test/scala/whisk/core/database/UserCommandTests.scala

```diff
@@ -68,6 +68,35 @@ class UserCommandTests extends FlatSpec with WhiskAdminCliTestBase {
     resultOk("user", "get", subject) shouldBe generatedKey
   }
 
+  it should "force update an existing user" in {
+    val subject = newSubject()
+    val oldKey = resultOk("user", "create", "--force", subject)
```

Comment:
is this `--force` intended?

Reply:
The idea is to tell "if user is not existing, enabling force should behave the same as without it".
Not sure if it's necessary?

Reply:
to be clear: use create X fails if  X already exists, but --force allows it to succeed and updates the uuid and key. is my understanding correct? if so, yes i'm ok with that.

Reply:
@rabbah yes that's correct